crates/nu-protocol/src/module.rs

```diff
@@ -157,6 +157,17 @@ impl Module {
 
         match head {
             ImportPatternMember::Name { name, span } => {
+                // raise errors if user wants to do something like this:
+                // `use a b c`: but b is not a sub-module of a.
+                let errors = if !rest.is_empty() && self.submodules.get(name).is_none() {
+                    vec![ParseError::WrongImportPattern(
+                        format!("You can only use sub-sub names from sub modules, but `{}` is not a module", String::from_utf8_lossy(name)),
```

Comment:
I struggle to find a better message but to me this seems very technical where a user doesn't know how to fix it.

Reply:
Hmm..What about this?
```nushell
❯ use foo c d
Error: nu::parser::wrong_import_pattern

  × Wrong import pattern structure.
   ╭─[entry #2:1:1]
 1 │ use foo c d
   ·           ┬
   ·           ╰── Trying to import something but the parent `c` is not a module, maybe you want to try `use <module> [<name1>, <name2>]`
   ╰────
```
Which guides user to `use module [name1, name2]`

Reply:
Good call!